filesec/file_security.go

```diff
@@ -69,6 +69,9 @@ type Config struct {
 
 	// DisableTLSVerify disables TLS verify in HTTP clients etc
 	DisableTLSVerify bool
+
+	//Support always overwriting the local filesystem cache
```

Comment:
space after `//` and it should start with the var name, so `// AlwaysOverwriteCache support always....`